cloud-service-providers/azure/azureml/cli/config.sh

```diff
@@ -22,21 +22,23 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 # AzureML Workspace and corresponding container registry related information
-subscription_id="<your-azure-subscription-id>"
-resource_group="<your-resource-group>"
-workspace="<your-azureml-workspace-name>"
-location="<your-azureml-region>" # eg: "southcentralus", "westeurope" etc.
+subscription_id="ab221ca4-f098-422d-ab2f-5073b3851e68"
+resource_group="NIMDeploDemo_rg"
+workspace="NIMDeployDemo_ws"
+location="southcentralus"
 
 # Azure keyvault creation related information
-ngc_api_key="<your-ngc-api-key>"
+ngc_api_key="azdwMG00YXNicWVzcnFrN3ZqMnFtcTAyb2U6NzRhZTUxODgtNzBhMS00ZDIzLThlODktZWNkNGQwMTE5OWRh"
```

Comment:
Can you remove the NGC API Key?

Reply:
I did not intend to check in the config.sh thanks for catching that up

Reply:
I reverted the changes